boxes/templates/boxes/box_list.html

```diff
@@ -62,120 +62,127 @@ <h3 class="box__title text text-darkest">{{box.name}}</h3>
+                {{ form.name.label_tag }}
+                {{ form.name }}
+              </p>
+              <div class="row">
+                <div class="col-xs-9 col-xxs-12 col-sm-9">
+                  <p>
+                    <span id="description-errors-js">{{ form.description.errors }}</span>
+                    {{ form.description.label_tag }}
+                    {{ form.description }}
+                  </p>
+                </div>
+                <div class="col-xs-3 col-xxs-12 col-sm-3">
+                  <p>
+                    <span id="max_messages-errors-js">{{ form.max_messages.errors }}</span>
+                    {{ form.max_messages.label_tag }}
+                    {{ form.max_messages }}
+                  </p>
+                  <br>
                 </div>
               </div>
-            </label>
-            <div class="row middle-xs center-xs">
-              <div class="col-xs-12">
-                <button type="submit" class="btn-default" value="">Create</button>
+              <div class="row bottom-xs padding-b-m">
+                <div class="col-xs-9 col-xxs-12 col-sm-8">
+                  <p>
+                    <span id="expires_at-errors-js">{{ form.expires_at.errors }}</span>
+                    {{ form.expires_at.label_tag }}
+                    {{ form.expires_at }}
+                  </p>
+                </div>
+                <div class="col-xs-3 col-xxs-12 col-sm-4">
+                  <div class="row middle-xs">
+                    <div class="col-xs-10 col-xxs-8 sm-10">
+                      <p><span id="never_expires-errors-js">{{ form.never_expires.errors }}</span>
+                        {{ form.never_expires.label_tag }}</p>
+                      </div>
+                      <div class="col-xs-2 col-xxs-4 sm-2">
+                        {{ form.never_expires }}
+                      </div>
+                    </div>
+                  </div>
+                </label>
               </div>
             </div>
+            <div class="col-xs-12">
+              <button type="submit" class="btn-default" value="">Create</button>
+            </div>
           </form>
         </div>
       </div>
-      <div class="md-overlay"></div>
     </div>
-  </div>
-  {% endblock content %}
+    <div class="md-overlay"></div>
+    {% endblock content %}
 
-  {% block scripts %}
-  {{ block.super }}
-  <script src="{% static 'javascripts/box_list.js' %}"></script>
-  <script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
-  <script src="{% static 'javascripts/box_create.js' %}"></script>
-  {% endblock scripts %}
+    {% block scripts %}
+    {{ block.super }}
+    <script src="{% static 'javascripts/box_list.js' %}"></script>
+    <script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
+    <script src="{% static 'javascripts/box_create.js' %}"></script>
+    <script type="text/javascript">
```

Comment:
We are trying to avoid inline javascript. Please move the code to `javascripts/box_create.js` or add this classes to the elements, since this seems to run every time.